src/bin/solang.rs

```diff
@@ -186,7 +186,7 @@ fn compile(compile_args: &Compile) {
     for filename in compile_args.package.get_input() {
         // TODO: this could be parallelized using e.g. rayon
         let ns = process_file(
-            filename,
+            &filename.canonicalize().unwrap(),
```

Comment:
Rather than canonicalizing the import paths and adjoining the filename to the import paths, it's better to canonicalize the filenames we _know_ exist in our host filesystem, and those are precisely the file names passed in by command line

Reply:
We cannot call `unwrap()` or `canonicalize()` because the file might not exist, it would at least need some better error handling.

Reply:
I agree on the unwrap, sorry sloppy coding on my end. We were calling canonicalize before I think, just somewhere else? My idea was to bubble up any canonicalization to locations where paths represent host file system paths rather than virtual file system paths.

Reply:
Wasn't sure what the best way to handle this was (since we don't return a Result). Just removed canonicalization since `process_file` tries to canonicalize anyway.